src/train/trainer/GradientBoostingRegressorTrainer/main.py

```diff
@@ -0,0 +1,21 @@
+from sklearn.ensemble import GradientBoostingRegressor
```

Comment:
This should be updated with the XGBoost model in future

Reply:
Could you update or create new trainer with that?
related issue: https://github.com/sustainable-computing-io/kepler-model-server/issues/89